main.py

```diff
@@ -1,7 +1,38 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
```

Comment:
Unpack  the actor tuple into `first_name` and `last_name` directly in the loop. This makes the code more readable and eliminates the need for indexing when accessing the elements of the actor tuple.

Reply:
one more time